texar/torch/utils/dtypes.py

```diff
@@ -96,6 +87,16 @@ def is_str(x):
     return isinstance(x, str)
 
 
+def get_supported_scalar_types():
```

Comment:
Oh, I thought we were only going to support a part of these types. For instance, `np.str` doesn't seem like it should be in `ScalarData`.

Reply:
Resolved in 0758eb5